js/index.js

```diff
@@ -154,10 +157,29 @@ const removeBook = (idx) => {
 };
 
 const main = () => {
+  contactForm.style.display = 'none';
   recreateUI();
 };
 
-main();
+const showContact = () => {
+  contactForm.style.display = 'block';
+  const existingBooks = document.querySelector('.books-container');
+  if (existingBooks) {
+    existingBooks.remove();
+  }
```

Comment:
* Nice work with the design 👏 but I would suggest you use the wireframe as a guide and ensure when you click on any of the links, the main area changes to display only one of the three content sections so for add book section you can make it change the main page area
* Also To change the contents of the page, don't use any library or framework, use only plain JavaScript and CSS, I can see you have used the bootstrap library [Check requirements here ](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website_v1_1.md#:~:text=Use%20this%20wireframe,JavaScript%20and%20CSS.)

Reply:
Noted on the first point. But on the second point, we were told not to use any _Javascript_ library or framework. We did not use any javascript library or framework. We only used bootstrap for the styling, and doesn't have anything to do with javascript. We got our past two pull requests approved while using bootstrap, so I believe it's allowed in this project. Let us know if we've made a mistake.

Reply:
@Davidosky007